### DISCUSSION TITLE
Welcome to General-VBA Discussions!

### DISCUSSION THREAD

**Discussion by ViperSRT3g** [ViperSRT3g/VBA-General#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
i want to copy the code from text file into the "Thisoutlooksession" module in outlook through vba script ,i am not able to find the way . 
is there any way to do it ?

**Reply by KallunWillock** [reply]
Unfortunately, I don't think it's possible. From what I understand, Outlook doesn't have the VBA Extensibility functionality. Do you have a lot of modules, etc. that you want to import?
